types/web-element.d.ts

```diff
@@ -159,7 +159,7 @@ export interface ScopedElement extends Element, PromiseLike<WebElement> {
 
   setAttribute(name: string, value: string | null): Promise<WebElement>;
 
-  takeScreenshot(shouldBeInView?: boolean): ElementValue<string>;
```

Comment:
Can we add a test for this as well

Reply:
Added. Not sure why I missed it last time.